server.js

```diff
@@ -3414,27 +3414,55 @@ cache(function(data, match, sendBadge, request) {
   var repo = match[2];  // eg, subtitleedit
   var version = match[3];  // eg, 3.4.7
   var format = match[4];
-  var apiUrl = githubApiUrl + '/repos/' + user + '/' + repo + '/compare/' + version + '...master';
   var badgeData = getBadgeData('commits since ' + version, data);
-  if (badgeData.template === 'social') {
-    badgeData.logo = getLogo('github', data);
-  }
-  githubAuth.request(request, apiUrl, {}, function(err, res, buffer) {
-    if (err != null) {
-      badgeData.text[1] = 'inaccessible';
-      sendBadge(format, badgeData);
-      return;
-    }
-    try {
-      var data = JSON.parse(buffer);
-      badgeData.text[1] = metric(data.ahead_by);
-      badgeData.colorscheme = 'blue';
-      sendBadge(format, badgeData);
-    } catch(e) {
-      badgeData.text[1] = 'none';
-      sendBadge(format, badgeData);
+
+  function setCommitsSinceBadge(user, repo, version) {
+    var apiUrl = githubApiUrl + '/repos/' + user + '/' + repo + '/compare/' + version + '...master';
+    if (badgeData.template === 'social') {
+      badgeData.logo = getLogo('github', data);
     }
-  });
+    githubAuth.request(request, apiUrl, {}, function (err, res, buffer) {
+      if (err != null) {
+        badgeData.text[1] = 'inaccessible';
+        sendBadge(format, badgeData);
+        return;
+      }
+      try {
+        var data = JSON.parse(buffer);
+        badgeData.text[1] = metric(data.ahead_by);
+        badgeData.colorscheme = 'blue';
+        badgeData.text[0] = getLabel('commits since ' + version, data);
+        sendBadge(format, badgeData);
+      } catch (e) {
+        badgeData.text[1] = 'none';
+        sendBadge(format, badgeData);
+      }
+    });
+  }
+
+  if(version.toLowerCase() === 'latest') {
```

Comment:
Let's just check `version === 'latest'`. If the badge doesn't work, people will know to fix it. If we allow this now we'll have to keep allowing it forever, or risk breaking things.

Reply:
ok ok. Got that 👍